sci-libs/symmetrica/symmetrica-2.0.ebuild

```diff
@@ -1,29 +1,21 @@
-# Copyright 1999-2014 Gentoo Foundation
+# Copyright 1999-2015 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Header: $
 
-EAPI=2
+EAPI=5
 
 inherit versionator
 
 DESCRIPTION="A collection of routine to handle a variety of topics"
 HOMEPAGE="http://www.neu.uni-bayreuth.de/de/Uni_Bayreuth/Fakultaeten/1_Mathematik_Physik_und_Informatik/Fachgruppe_Informatik/prof_diskrete_algorithmen/en/research/SYMMETRICA/index.html"
-MY_P=SYM$(replace_all_version_separators '_')
+MY_P=SYM${PV//./_}
```

Comment:
This was a hint by @junghans (second comment on the outdated diff). Should I revert this or let it as is and additionally remove the inherit versionator (which I forgot to clean up).